api/print.php

```diff
@@ -188,7 +189,8 @@
 // print image
 $status = 'ok';
 $linecount = 0;
-$cmd = sprintf($config['commands']['print'], $vars['printFile']);
```

Comment:
Before check 
config max_multi === 1 and define arg1 and arg2 which will give proper arg handling